Dockerfile/entrypoint.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-. /opt/ros/"${ROS_DISTRO}"/setup.sh
+. /opt/ros/rolling/setup.sh
```

Comment:
do you mind to set a env in the dockerfile ? instead of editing this file?

```
ENV ROS_DISTRO rolling
```

Reply:
I was thinking of doing that, but since rolling was hardcoded everywhere I decided to keep it the same way as throughout whole Dockerfile. I will add the env and make appropriate changes soon.